drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -242,10 +242,10 @@ static void i2c_standard_port_timing_regs_400khz(uint8_t port)
 	/* Port clock frequency depends on setting of timing registers. */
 	IT83XX_SMB_SCLKTS(port) = 0;
 	/* Suggested setting of timing registers of 400kHz. */
-	IT83XX_SMB_4P7USL = 0x6;
+	IT83XX_SMB_4P7USL = 0x3;
```

Comment:
Are these documented somewhere? A quick search on ITE's site I didn't see any docs on what these mean or what values should be here. I'm curious why the suggest values are changing here, if its to correcting for a board specific problem this isn't the place to do that.

Reply:
<img width="559" alt="timing_reg" src="https://user-images.githubusercontent.com/73827039/154918790-a4c08e5f-a0a7-4b33-976c-2ad020d06b4a.png">
<img width="620" alt="timing_reg2" src="https://user-images.githubusercontent.com/73827039/154918799-d3ff7013-9db8-4c50-ae93-4afa0d4e61a5.png">

As shown above, we can control the TSU_STA and THD_DAT simultaneously by changing the value of the register 4P7USL. Likewise, we can control the tsu_dat by changing the value of the register 250NS as well.

The lower the value of 4P7USL is set, the shorter the THD_DAT we will have. Likewise, the higher we configure the 250NS, the longer the tsu_dat will be. Therefore, we can configure the 4P7USL and the 250NS properly so that we can keep the Tlow unchanged.

Reply:
Thank you for the great explanation, that makes perfect sense as to what these values are doing now.